pallets/asset/src/lib.rs

```diff
@@ -2015,7 +2015,7 @@ impl<T: Trait> Module<T> {
         // TODO: Once gas estimate function will be introduced, arbitrary gas value will be replaced by the estimated gas
         let (res, _gas_spent) =
             Self::call_extension(extension_caller, dest, GAS_LIMIT, encoded_data);
```

Comment:
The benchmark should consume 100% of this `GAS_LIMIT`. How much is it using right now?

Reply:
12424000000